tee/prototype/enclave_internal_test.go

```
@@ -126,6 +134,63 @@ func TestEnclave(t *testing.T) {
 		requiree.NoError(err)
 	}
 
+	doWith := func(bp *tee.BalanceProof, aliceDo, bobDo cltest.ClientAction) {
+		switch bp.Balance.Account.String() {
+		case aliceAd.Address.String():
+			requiree.NoError(aliceDo(ctx, bp))
+		case bobAd.Address.String():
+			requiree.NoError(bobDo(ctx, bp))
+		default:
+		}
+	}
+
+	t.Log("Sending two exit TXs.")
+	for _, bp := range bps {
+		doWith(bp, alice.Exit, bob.Exit)
+	}
+
+	// Signalling the enclave to stop now, so that it doesn't start new epochs on
+	// the next block.
+	t.Log("Set Enclave to shutdown after next phase.")
+	enc.Shutdown()
+
+	seal("exitPhase", 1)
+
+	t.Log("Sending two withdrawal TXs.")
+	for _, bp := range bps {
+		doWith(bp, alice.Withdraw, bob.Withdraw)
+	}
+
+	aliceNewBal, _ := setup.SimBackend.BalanceAt(ctx, aliceAd.Address, nil)
+	bobNewBal, _ := setup.SimBackend.BalanceAt(ctx, bobAd.Address, nil)
```

Why do you ignore the errors instead of `requiree.NoError`?

---

I thought this error would be caught anyway, when checking the balances. But I realize this is only by "chance" so better add those checks back.